wallet-connect/Cargo.toml

```diff
@@ -14,6 +14,7 @@ ethers = { version = "0.17", features = ["rustls"] }
 eyre = "0.6"
 futures = "0.3"
 hmac = "0.12"
+open = "3.2"
```

Comment:
```suggestion
open = "3"
```

Reply:
The example path can not simply specify a library path, since it is managed by CMake instead of gcc. One approach is generating the database first and check lint after build.

Reply:
Or we can generate database before building..

Reply:
Ok... I checked, it is not appropriate to check lint before the demo project correctly setup (e.g. before the bindings headers are ready).